mods/persistence/modules/mob/living/carbon/human/human.dm

```diff
@@ -31,12 +31,12 @@
 	. = ..()
 	LATE_INIT_IF_SAVED
 
-/decl/species/create_organs(var/mob/living/carbon/human/H)
-	//We don't want to delete the organs we loaded from the save
-	if(!H.persistent_id)
-		. = ..()
+// /decl/species/create_organs(var/mob/living/carbon/human/H)
```

Comment:
Unless you have a specific reason, deleting these lines is preferable to commenting them out.

Reply:
I commented them because I need to remember about them when I apply the organ fix pr to this.